text/src/autogluon/text/text_prediction/predictor/predictor.py

```diff
@@ -422,7 +422,10 @@ def predict_proba(self, data, as_pandas=True, as_multiclass=True):
                 index = data.index
             else:
                 index = None
-            output = pd.DataFrame(output, index=index)
+            if output.ndim == 1:
+                output = pd.Series(output, index=index)
```

Comment:
Add documentation that this can return Series, documentation only mentions DataFrame. (Consider re-using Tabular's documentation for predict_proba)

Reply:
Added in the docstring of `as_pandas`